enterprise/reporting/reporting.go

```diff
@@ -139,6 +139,20 @@ func (r *DefaultReporter) DatabaseSyncer(c types.SyncerConfig) types.ReportingSy
 	if err != nil {
 		panic(fmt.Errorf("could not run reports migrations: %w", err))
 	}
+	if config.IsSet("Reporting.autoVacuumCostLimit") {
```

Comment:
- We can execute the migrator regardless for this option.
- We can pass config in the template context
- The template itself can have an if clause checking the existence of the config option (yes go templates can call functions)

Reply:
Made the change